### ISSUE TITLE
Switch to RapidYAML

### ISSUE THREAD

**Issue by SirNate0** [getml/reflect-cpp#182]
If speed of yaml parsing/emitting is a concern (as suggested by the [benchmarks](https://github.com/getml/reflect-cpp/tree/main/benchmarks)), consider switching to [Rapid YAML](https://github.com/biojppm/rapidyaml) instead of yaml-cpp.

**Comment by liuzicheng1987** [comment]
Yes, someone on Reddit suggested that as well. I think it is a good idea.

**Comment by liuzicheng1987** [comment]
@SirNate0, do you have any experience with RapidYAML's API? The documentation is not too great and it's not very intuitive.

**Comment by MirrasHue** [comment]
@liuzicheng1987 Hey, have you contacted the developer of rapid yaml? Maybe he has some interest in helping you out
